### PR TITLE
#324 Redesign the schedule week view

### PR DESCRIPTION
**Pull request by Ymkavi**
image

### CODE REVIEW

File: frontend/src/app/modules/availability/new-slot/schedule-week/schedule-week.component.sass [hunk 1]
```diff
@@ -1,2 +1,25 @@
-.wrapper
+@import 'src/styles/color-variables.sass'
+@import 'src/styles/screen-variables.sass'
+
+.schedule-calendar-wrapper
     max-height: 60vh
+    .cal-week-view .cal-time-events .cal-event
+        width: calc(100% - 60px)
```
Comment (Erbidium) [inline frontend/src/app/modules/availability/new-slot/schedule-week/schedule-week.component.sass]: Maybe we can change layout in order not to do such calculations? It doesn't look good

File: frontend/src/app/modules/availability/new-slot/schedule-week/schedule-week.component.sass [hunk 2]
```diff
@@ -1,2 +1,25 @@
...
+        margin-left: 30px
+    .cal-week-view .cal-time-events .cal-resize-handle
+        width: calc(100% - 50px)
```
Comment (Erbidium) [inline frontend/src/app/modules/availability/new-slot/schedule-week/schedule-week.component.sass]: Same

File: frontend/src/app/modules/availability/new-slot/schedule-week/schedule-week.component.sass [hunk 3]
```diff
@@ -1,2 +1,25 @@
...
+        margin-left: 25px
+        border-radius: 4px
+        background-color: $raven
+        height: 5px
+    .cal-event
+        background-color: $white-color
+        border-color: $button-border
+    @media screen and (max-width: $desktop-min-width)
+        .cal-week-view .cal-time-events .cal-event
+            width: calc(100% - 40px)
```
Comment (Erbidium) [inline frontend/src/app/modules/availability/new-slot/schedule-week/schedule-week.component.sass]: Same

File: frontend/src/app/modules/availability/new-slot/schedule-week/schedule-week.component.sass [hunk 4]
```diff
@@ -1,2 +1,25 @@
...
+            margin-left: 20px
+        .cal-week-view .cal-time-events .cal-resize-handle
+            width: calc(100% - 28px)
```
Comment (Erbidium) [inline frontend/src/app/modules/availability/new-slot/schedule-week/schedule-week.component.sass]: Same

**Review by Erbidium — CHANGES_REQUESTED** [review]

**Review by tatianahutii — APPROVED** [review]

**Review by Erbidium — APPROVED** [review]

**Review by apofis96 — APPROVED** [review]

**Pull request merged**